aip/general/0203.md

```diff
@@ -54,6 +54,10 @@ A field **should** only be described as required if _either_:
   provided as part of the request, and failure to do so **must** cause an error
   (usually `INVALID_ARGUMENT`).
 
+**Note:** We define the term "truthy" as follows: For primitives, repeated
```

Comment:
I think it would be better to define this in a way that doesn't require Python knowledge. I don't mind a reference to Python as a sort of "this is equivalent to" but readers shouldn't have to start reading Python docs in order to understand the AIP.

Reply:
Fair point. Done.